### PR TITLE
Add CMS Guides landing page

### PR DESCRIPTION
**Pull request by Jutanium**

### CODE REVIEW

File: src/pages/en/guides/cms.md
Comment (sarah11918) [inline src/pages/en/guides/cms.md:31]:
```suggestion
Some CMSes, like Storyblok, provide an Astro [integration](/en/guides/integrations-guide/) that helps fetch the content specifically for an Astro site. Others provide a JavaScript SDK, a library that you install and use to fetch your remote content.

## Can I use Astro without a CMS?

Yes! Astro provides built-in ways to [author content](/en/guides/content/), including support for Markdown pages.
```

File: src/pages/en/guides/cms.md
Comment (sarah11918) [inline src/pages/en/guides/cms.md:9]:
```suggestion
**Ready to connect a Headless CMS to your Astro project?** Follow one of our guides to integrate a CMS.
```

File: src/pages/en/guides/cms.md
Comment (sarah11918) [inline src/pages/en/guides/cms.md:18]:
```suggestion

It's up to you to determine what content on your site is managed by the CMS, and where on your site you want to display it.
```
Did you want this to be its own paragraph?

File: src/pages/en/guides/cms.md
Comment (sarah11918) [inline src/pages/en/guides/cms.md:20]:
```suggestion
You might use a CMS to build a blog for someone who doesn't know Markdown — they can write posts using a CMS's rich text editor instead. If you're building an ecommerce shop, you might use a CMS to organize and edit shop items. You can configure your CMS to hold certain pieces of information about each shop item, even making them required to avoid incomplete listings, and your Astro project can then fetch and display them.
```
"publish" shop items felt a little weird -- jargony? Maybe something more related to "organizing" or "maintaining"?

Also tried to think more user-y re: shop items?

File: src/pages/en/guides/cms.md
Comment (sarah11918) [inline src/pages/en/guides/cms.md:25]:
```suggestion
Because Astro takes care of the _presentation_ of your content, you'll want to choose a _headless_ CMS, like those in the list below. This means that the CMS helps you write your content, but doesn't generate a site that displays it. Instead, you fetch the content data hosted there and use in your Astro project. 
```
I mean, depends on what the page looks like! I figured, like deploy, the content would come BEFORE all the icons.  😅

**Review by sarah11918 — COMMENTED** [review]
Looks great so far Dan!

File: src/pages/en/guides/cms.md
Comment (Jutanium) [reply src/pages/en/guides/cms.md:9]: 🤦

File: src/pages/en/guides/cms.md
Comment (Jutanium) [reply src/pages/en/guides/cms.md:24]: On deploy the icons come first! I don't think we want them to scroll through the content first

**Review by sarah11918 — APPROVED** [review]

**Comment by kevinzunigacuellar** [comment]
This looks great! ⚡️🚀

**Comment by sarah11918** [comment]
Beautiful work, @delucis! 

I think we discussed a custom ordering for these, which really only affects Storyblok, which is a larger partner. (Contentful and Butter are the other ones, but they're coincidentally already at the beginning.) I don't think this is blocking, but we *might* want Storyblok first eventually?

Also, just checking also with @Jutanium - is the plan to merge this WITH the Storyblok full guide draft merging? Or are we merging with the stub? If with the stub, we might want to edit the stub to say "full guide dropping soon" vs "help us create this".

**Comment by sarah11918** [comment]
Heads up that the paragraphs are looking a little long, so I'm going to take a pass at the content as a whole. Don't merge without me!!  😄

**Comment by delucis** [comment]
Won't merge without you! Storyblok will move up the order once it's no longer a stub after #1944.

**Comment by sarah11918** [comment]
Honestly, with Butter and Contentful being the only other two we'd *want* to move up, we lucked out! No other changes needed, I don't think!  😄

File: src/pages/en/guides/cms.md
Comment (sarah11918) [inline src/pages/en/guides/cms.md:24]:
```suggestion
A Content Managament System lets you write content and manage assets outside of your Astro project.

This unlocks new features for working with content. Most CMSes give you a visual content editor, the ability to specify standard types of content, and a way to collaborate with others. 

A CMS can be useful for content that follows a particular structure, often giving you a dashboard-like experience and WYSIWYG editing tools. You can use a CMS to write blog posts using a CMS's rich text editor instead of Markdown files. You can also use a CMS to maintain product listings for an eCommerce shop, even making certain fields required to avoid incomplete listings. 

Your Astro project can then fetch and display your content from the CMS, wherever on your site you want to display it.
```

Thoughts?

**Comment by sarah11918** [comment]
"wherever AND HOWEVER on your site you want to display it."  ???

**Comment by sarah11918** [comment]
@delucis - I think we're coming to the conclusion that we'd like to be able to specify a custom order for the non-stub pages!

File: src/components/CMSGuidesNav.astro
Comment (delucis) [inline src/components/CMSGuidesNav.astro:16]: We can update this array if we ever want to move specific pages up to the top of the nav list.

Comment (sarah11918) [reply src/components/CMSGuidesNav.astro:16]: Perfect!

File: src/pages/en/guides/cms.md
Comment (sarah11918) [inline src/pages/en/guides/cms.md:26]:
```suggestion
Your Astro project can then fetch your content from your CMS and display it, wherever and however you want on your site.
```
Do we like this? @Jutanium

Comment (Jutanium) [reply src/pages/en/guides/cms.md:26]: It isn't clear what the different "however" possibilities are within "displaying" it - how about:

> Your Astro project can then fetch your content from your CMS and use it, wherever and however you want on your site.

**Comment by sarah11918** [comment]
OK, just checking in with everyone: I *think* this is good to go, as soon as we've merged Storyblok Guide? #1944

**Comment by delucis** [comment]
I’m happy!

**Pull request merged**
